### PR TITLE
Porting btcd to nav first commits

### PR DESCRIPTION
**Pull request by aguycalled**
Adjustment of name from btcd to navd, and first network adjustments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,44 +80,44 @@ Install a newer MSI
...
+$ ./navd
 ```
 
 ## IRC
 
 - irc.freenode.net
-- channel #btcd
-- [webchat](https://webchat.freenode.net/?channels=btcd)
+- channel #navd
+- [webchat](https://webchat.freenode.net/?channels=navd)
```
Comment (matt-auckland) [inline README.md:112]: pffft we don't have irc 😛

Comment (aguycalled) [reply README.md:112]: we should, missing terminal clients :p

File: blockchain/chainio.go [hunk 2]
```diff
@@ -779,7 +779,7 @@ func deserializeUtxoEntry(serialized []byte) (*UtxoEntry, error) {
 }
 
 // dbFetchUtxoEntry uses an existing database transaction to fetch all unspent
-// outputs for the provided Bitcoin transaction hash from the utxo set.
+// outputs for the provided Navcoin transaction hash from the utxo set.
 //
```
Comment (matt-auckland) [inline blockchain/chainio.go]: `Navcoin` -> `NavCoin`. There are a few lines like this (if we care)

**Review by zanuka — APPROVED** [review]

File: blockchain/doc.go [hunk 3]
```diff
@@ -70,7 +70,7 @@ violations through type assertions.  In addition, callers can programmatically
 determine the specific rule violation by examining the ErrorCode field of the
 type asserted blockchain.RuleError.
 
-Bitcoin Improvement Proposals
+Navcoin Improvement Proposals
```
Comment (matt-auckland) [inline blockchain/doc.go]: NavCoin Improvement Proposal doesn't lend itself to a very politically correct acronym 💦 

Pehraps NavCoin Enchanement Proposal would be better (NEP)

File: blockchain/doc.go [hunk 4]
```diff
@@ -70,7 +70,7 @@ violations through type assertions.  In addition, callers can programmatically
...
 
 This package includes spec changes outlined by the following BIPs:
```
Comment (matt-auckland) [inline blockchain/doc.go]: BIP -> NEP ?

**Review by matt-auckland — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (zanuka) [reply README.md:112]: agree

**Pull request merged**